filament/backend/src/opengl/OpenGLDriver.cpp

```diff
@@ -1281,12 +1281,17 @@ void OpenGLDriver::createRenderTargetR(Handle<HwRenderTarget> rth,
 
     rt->gl.samples = samples;
 
+    auto valueForLevel = [](size_t level, size_t value) {
+        return std::max(size_t(1), value >> level);
```

Comment:
Could you have an underflow here in some cases?

Reply:
meaning?